scripts/debian/changelog

```diff
@@ -0,0 +1,95 @@
+nova-agent (2.1.23) unstable; urgency=low
```

Comment:
Won't the version be 2.1.23-1

Reply:
With '2.1.23-1', the '-1' would indicate a source package.  As it is a binary/native package it is just '2.1.23'.